pkg/manager/impl/node_execution_manager.go

```diff
@@ -4,6 +4,9 @@ import (
 	"context"
 	"strconv"
 
+	"github.com/gogo/protobuf/proto"
```

Comment:
can we use `github.com/golang/protobuf/proto` instead? gogo behaves weirdly with oneofs

Reply:
Sure. Never paid attention for that as intellij did that for me.